plugins/services/src/js/structs/__tests__/Application-test.js

```diff
@@ -539,20 +539,32 @@ describe("Application", function() {
           // DCOS_SERVICE_SCHEME: 'https'
         }
       });
-      expect(service1.getWebURL()).toEqual(null);
-      expect(service2.getWebURL()).toEqual(null);
-      expect(service3.getWebURL()).toEqual(null);
+      expect(service1.getWebURL()).toEqual("");
+      expect(service2.getWebURL()).toEqual("");
+      expect(service3.getWebURL()).toEqual("");
     });
 
-    it("returns null if the name is an empty string", function() {
+    it("returns empty string if the name is an empty string", function() {
       const service = new Application({
         labels: {
           DCOS_SERVICE_NAME: "",
           DCOS_SERVICE_PORT_INDEX: "80",
           DCOS_SERVICE_SCHEME: "https"
         }
       });
-      expect(service.getWebURL()).toEqual(null);
+      expect(service.getWebURL()).toEqual("");
+    });
+
+    it("returns empty string if service is SDK Service", function() {
```

Comment:
description and test don't match here :)

Reply:
uh, will fix 👍